_posts/2019-04-02-addhoc.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: post
+title: "Making React HOC functions the easy way with addhoc"
+date: 2019-04-02 09:00:00 -0700
+cover: /assets/images/headers/addhoc-cover.jpg
+excerpt: As defined in the React documentation, a higher-order component, or HOC, is a function that returns a React component that wraps a specified child component and often provides augmented functionality. Implementing HOCs can be hard when considering hoisting statics, managing ref forwarding, and handling display name. addhoc is a newly released open-source package that aims to handle these challenges for you.
```

Comment:
What does this mean?

> Implementing HOCs can be hard when... handling display name.

Is there a grammar change or some alternative phrasing that might provide more context for "handling display name"?

Reply:
It's further covered in the article -- this is just an excerpt.